src/home/positions/index.tsx

```diff
@@ -154,15 +190,59 @@ const Positions = () => {
               sx={{ marginLeft: '32px' }}
               label={
                 <Typography variant="h6">
-                  <FormattedMessage description="terminatedPositions" defaultMessage="Closed positions" />
+                  <FormattedMessage description="emptyPositions" defaultMessage="Empty" />
                 </Typography>
               }
             />
+            <StyledTab
+              disableRipple
+              sx={{ marginLeft: '32px' }}
+              label={
+                <Typography variant="h6">
+                  <FormattedMessage description="terminatedPositions" defaultMessage="Closed" />
+                </Typography>
+              }
+            />
+            {!!permissionedPositionsInProgress.length && (
+              <StyledTab
+                disableRipple
+                sx={{ marginLeft: '32px' }}
+                label={
+                  <Typography variant="h6">
+                    <FormattedMessage description="terminatedPositions" defaultMessage="Permissioned" />
+                  </Typography>
+                }
+              />
+            )}
           </StyledTabs>
         </StyledTabsContainers>
         <StyledPositionsContainer>
           <StyledPaper variant="outlined">
-            {tabIndex === 0 ? <CurrentPositions isLoading={!hasLoadedPositions || isLoading} /> : <History />}
+            {(() => {
```

Comment:
I think that an inline switch statement adds clarity to the tabs, so I decided to use an inline IIFE instead of inline jsx like
```
{tabIndex === 0 && <PositionList....>}
{tabIndex === 1 && <PositionList...}
```

but its just a syntax preference so I'm ok refactoring to the above example if you prefer, or however you believe it's clearer.